samples/features/reporting-services/ssrs-migration-rss/ssrs_migration.rss

```diff
@@ -948,21 +948,27 @@ End Function
 
 'Helper function to construct correctly formatted filename
 Function GetSnkFilename(srcFilename As String, type As String) As String
-	Dim hasFE As Boolean = HasFileExtension(srcFilename)
-	If SnkIsNative And hasFE And Not GetExtension(type) = "" Then 'Cut off file extension, if present            
+	Dim currentFE As String = GetFileNameExtension(srcFilename)
```

Comment:
Please avoid abbreviations FE should be FileExtension

Reply:
Corrected.